flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "tybalt - A library for creating web components";
+  inputs = {
+    nixpkgs = { url = "github:NixOS/nixpkgs/release-23.05"; };
+    turbo = {
+      url = "github:alexghr/turborepo.nix/v1.10.12";
```

Comment:
@doug-wade I think we can improve this section. The version is wrong here, it should match the same version of Turbo as on the `package.json` file.